From: Andrea Bolognani <eof@kiyuko.org>
Date: Sun, 9 Sep 2018 18:27:47 +0200
Subject: Install more files

Running 'make install' should result in not only the application
and corresponding library, but also the release notes, license and
most importantly configuration file being installed.

Bug: https://github.com/conformal/spectrwm/pull/220
---
 linux/Makefile | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/linux/Makefile b/linux/Makefile
index 7ceca3b..1667ff5 100644
--- a/linux/Makefile
+++ b/linux/Makefile
@@ -1,8 +1,10 @@
 PREFIX       ?= /usr/local
 BINDIR       ?= $(PREFIX)/bin
 LIBDIR       ?= $(PREFIX)/lib
+SYSCONFDIR   ?= $(PREFIX)/etc
 DATAROOTDIR  ?= $(PREFIX)/share
 MANDIR       ?= $(DATAROOTDIR)/man
+DOCDIR       ?= $(DATAROOTDIR)/doc/spectrwm
 XSESSIONSDIR ?= $(DATAROOTDIR)/xsessions
 
 BUILDVERSION    = $(shell sh $(CURDIR)/../buildver.sh)
@@ -49,14 +51,19 @@ clean:
 install: all
 	install -m 755 -d $(DESTDIR)$(BINDIR)
 	install -m 755 -d $(DESTDIR)$(LIBDIR)
+	install -m 755 -d $(DESTDIR)$(SYSCONFDIR)
 	install -m 755 -d $(DESTDIR)$(MANDIR)/man1
+	install -m 755 -d $(DESTDIR)$(DOCDIR)
 	install -m 755 -d $(DESTDIR)$(XSESSIONSDIR)
 	install -m 755 spectrwm                    $(DESTDIR)$(BINDIR)
 	ln -sf spectrwm                            $(DESTDIR)$(BINDIR)/scrotwm
 	install -m 644 libswmhack.so.$(LIBVERSION) $(DESTDIR)$(LIBDIR)
 	ln -sf libswmhack.so.$(LIBVERSION)         $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBMAJORVERSION)
 	ln -sf libswmhack.so.$(LIBVERSION)         $(DESTDIR)$(LIBDIR)/libswmhack.so
-	install -m 644 ../spectrwm.1               $(DESTDIR)$(MANDIR)/man1/spectrwm.1
+	install -m 644 ../spectrwm.conf            $(DESTDIR)$(SYSCONFDIR)
+	install -m 644 ../spectrwm.1               $(DESTDIR)$(MANDIR)/man1
+	install -m 644 ../CHANGELOG.md             $(DESTDIR)$(DOCDIR)
+	install -m 644 ../LICENSE.md               $(DESTDIR)$(DOCDIR)
 	install -m 644 spectrwm.desktop            $(DESTDIR)$(XSESSIONSDIR)
 
 uninstall:
@@ -65,7 +72,10 @@ uninstall:
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBVERSION)
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so.$(LIBMAJORVERSION)
 	rm -f $(DESTDIR)$(LIBDIR)/libswmhack.so
+	rm -f $(DESTDIR)$(SYSCONFDIR)/spectrwm.conf
 	rm -f $(DESTDIR)$(MANDIR)/man1/spectrwm.1
+	rm -f $(DESTDIR)$(DOCDIR)/CHANGELOG.md
+	rm -f $(DESTDIR)$(DOCDIR)/LICENSE.md
 	rm -f $(DESTDIR)$(XSESSIONSDIR)/spectrwm.desktop
 
 .PHONY: all clean install uninstall
